UKHomeOffice/sgmr-service
# SPCR-527 Change 6hr time band to 2hr

### AC / Description of changes
Change the 6hr time band to 2hr on both arrival and departure pages


### To Test
- create a voyage
- enter a start time and end time with more than 2hrs difference
- click save and continue
- > form should error
- change to a difference 2hrs or less
- click save and continue
- > form should not error

Do this for both arrival and departure


### Notes

---

**Remember to merge to feature branches not `master`.**
